uint/src/uint.rs

```diff
@@ -1130,6 +1165,12 @@ macro_rules! construct_uint {
 			}
 		}
 
+		impl_try_from_for_primitive!($name, u8);
+		impl_try_from_for_primitive!($name, u16);
+		impl_try_from_for_primitive!($name, u32);
+		impl_try_from_for_primitive!($name, usize);
+		impl_try_from_for_primitive!($name, u64);
```

Comment:
Is it useful to impl these for `i8`…`isize`,`i64` as well?

Reply:
`i8` ... `i64`, `i128` impl added :]